include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -77,6 +77,8 @@ template <typename T>
 #ifdef ORT_API_MANUAL_INIT
 const OrtApi* Global<T>::api_{};
 inline void InitApi() { Global<void>::api_ = OrtGetApiBase()->GetApi(ORT_API_VERSION); }
+inline void InitApi(const OrtApi* api) { Global<void>::api_ = api; }
+inline bool ApiIsInit() { return Global<void>::api_ != nullptr; }
```

Comment:
Need to discuss the best way to handle this. See: https://github.com/microsoft/onnxruntime/pull/12998#discussion_r986236900